template.rb

```diff
@@ -64,11 +67,16 @@ def add_spec_gems
 end
 
 def copy_solidus_starter_frontend_files
+  remove_file 'app/views/layouts/application.html.erb'
```

Comment:
The template now forcefully removes any application layout from the Rails app in order to replace it with the application layout from Solidus.

Reply:
Yep, I'll remove this `remove_file` call once I rename the layout to `storefront`. TODO.